docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+---
+version: "3"
+services:
+  pmm-server:
+    container_name: pmm-server
+    image: perconalab/pmm-server:dev-latest
+    volumes:
+      - "../grafana/public:/usr/share/grafana/public"
+    ports:
+      - 80:80
+      - 443:443
+    environment:
+      - PMM_DEBUG=1
+      - PERCONA_TEST_SAAS_HOST=check-dev.percona.com:443
+      - PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
```

Comment:
We also need `ENABLE_DBAAS=1` I guess

Reply:
I think there is no need for it as we don't execute dbaas tests here, and we have an option in settings now.